bump-tag

```diff
@@ -1,66 +1,268 @@
 #!/bin/bash
 
-set -e
```

Comment:
Maybe we still want set -e?

Reply:
This has been debated many times (in different contexts for different scripts :) ) internally. One reason for not using it is that the behavior becomes more unpredictable and of maybe more importance - makes the code harder to read.
But if someone would like to rewrite the code to make it possible to use `set -e`, then please go ahead since it will be up to the developers left at SUNET to decide what you like best.

Reply:
Good point!